src/agoradatatools/great_expectations/gx/expectations/ui_config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "data_asset_type": null,
+  "expectation_suite_name": "ui_config",
+  "expectations": [
+    {
+      "expectation_type": "expect_column_mostly_string_length",
+      "kwargs": {
+        "column": "columns",
+        "length_threshold": 0,
+        "operator": ">",
+        "target_field": "tooltip",
+        "valid_string_threshold": 0.35
```

Comment:
I'm not entirely sure what threshold to use here, but based on the validation rule, there are 109 dictionaries and 39 of them are considered valid. For now, I've set the threshold to 0.35 as a starting point.

Reply:
I'm not sure either, I think this is a question for @JessterB